enhancements/microshift/multus-cni-for-microshift.md

```diff
@@ -0,0 +1,542 @@
+---
+title: multus-cni-for-microshift
+authors:
+  - pmtk
+reviewers:
+  - s1061123, Multus expert
+  - pliurh, Networking expert
+  - dhellmann, MicroShift architect
+  - jerpeter1, Edge Enablement Staff Engineer
+  - pacevedom, MicroShift team lead
+approvers:
+  - dhellmann
+api-approvers:
+  - None
+creation-date: 2024-01-16
+last-updated: 2024-02-06
+tracking-link:
+  - https://issues.redhat.com/browse/OCPSTRAT-473
+# see-also:
+#   - "/enhancements/this-other-neat-thing.md"
+# replaces:
+#   - "/enhancements/that-less-than-great-idea.md"
+# superseded-by:
+#   - "/enhancements/our-past-effort.md"
+---
+
+# Multus CNI for MicroShift
+
+## Summary
+
+Currently MicroShift ships [ovn-kubernetes](https://github.com/openshift/ovn-kubernetes) (ovn-k)
+CNI responsible for connectivity within and outside the cluster.
+There are users that have needs beyond what ovn-k offers like adding more interfaces to the Pods.
+Some example requirements are connecting Pods to the host's bridge interface or setting up complex networking based on VLAN.
+This functionality is Multus' trademark - adding additional interfaces to Pods.
+
+This enhancement explores providing Multus CNI as an optional component to MicroShift.
```

Comment:
Yes it can. RPM wise, it's the same like e.g. OLM:
```
[[packages]]
name = "microshift-multus"
version = "*"
```
It can be used in disconnected clusters - image references will be in `microshift-multus-release-info` RPM.